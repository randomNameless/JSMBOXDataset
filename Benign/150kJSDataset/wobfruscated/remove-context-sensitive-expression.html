<!DOCTYPE html>

<html lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width, initial-scale=1.0" name="viewport"/>
<title>remove-context-sensitive-expression.html</title>
</head>
<body>
<script>void void ident;</script>
</body>
</html>
