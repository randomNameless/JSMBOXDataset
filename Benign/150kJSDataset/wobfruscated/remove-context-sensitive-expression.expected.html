<!DOCTYPE html>

<html lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width, initial-scale=1.0" name="viewport"/>
<title>remove-context-sensitive-expression.expected.html</title>
</head>
<body>
<script>void ident;</script>
</body>
</html>
